imageref.go

```diff
@@ -84,12 +84,7 @@ func (i *ImageRef) Export(outputFormat string) ([]byte, error) {
 	case Jpg:
 		result, _, err = i.ref.ExportJpeg(vips.NewJpegExportParams())
 	case WebP:
-		result, _, err = i.ref.ExportWebp(&vips.WebpExportParams{
-			StripMetadata:   false,
-			Quality:         75,
-			Lossless:        false,
-			ReductionEffort: 0,
-		})
+		result, err = i.ExportWebp(WebpExportParams{})
```

Comment:
here, the quality and reductionEffort will be 0, is that ok? this change will also affect the old preview-service, is that ok?

maybe pass a pointer in the method and if its nil, then use a default unexported value like:

```
	var defaultWebpParams = WebpExportParams{
		StripMetadata:   false,
		Quality:         75,
		Lossless:        false,
		ReductionEffort: 4,
	}
```

Reply:
oh, damn, no, I meant to fill this struct actually

Reply:
as long as we don't update the dependency the old preview service will work the same, bc of the hash in the mod file

Reply:
but I actually think this wont hurt there either, I will test wether increasing quality has any performance effect

Reply:
performance not really, but the size increases

Reply:
I updated it to 80 since it seems to improve quality without sacrificing much performance and size